File: src/main/java/com/wzg/scanner/inner/IScannerDeviceHelper.java
package com.wzg.scanner.inner;

/**
 * 可服用扩展
 */
public interface IScannerDeviceHelper {


    void registerScanReceiver();


    void unregisterScanReceiver();


    void registerScanListener(OnScanListener scanListener);


    interface OnScanListener {

        void onReceivedScanCode(String message);
    }


}

File: lib_scanner/src/main/java/com/blue/lib_scanner/inner/IScannerDeviceHelper.kt
package com.blue.lib_scanner.inner

/**
 * 可服用扩展
 */
interface IScannerDeviceHelper {
    fun registerScanReceiver()
    fun unregisterScanReceiver()
    fun registerScanListener(scanListener: OnScanListener?)
    interface OnScanListener {
        fun onReceivedScanCode(message: String?)
    }
}
